db-client/src/main/java/com/splicemachine/db/client/net/NetConnectionReply.java

```diff
@@ -2452,6 +2463,17 @@ protected int parseCODPNTDR() throws DisconnectException {
         return readBytes();
     }
 
+    // A generated token that the target SQLAM uses to interrupt the execution
+    // of a DDM command for a specific requester.
+    protected byte[] parseRDBINTTKN(boolean skip) throws DisconnectException {
+        parseLengthAndMatchCodePoint(CodePoint.RDBINTTKN);
```

Comment:
Why is `parseLengthAndMatchCodePoint` executed when `skip` input argument is `false` that leads to `return null`?